tests/test_embedding.py

```diff
@@ -19,8 +21,13 @@ def test_embedding_query_string(db: gp.Database):
         )
         .check_unique(columns={"id"})
```

Comment:
1. `check_unique` actually does more than *check", it creates indexes which is not obvious from the function name. Shall we consider to rename the function?
2. Need tests multi columns for `check_unique()` and `search()`. Another PR will be fine since it is not relevant to this one.

Reply:
```
[5.4.1. Check Constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS)
[5.4.2. Not-Null Constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-NOT-NULL)
[5.4.3. Unique Constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-UNIQUE-CONSTRAINTS)

5.4.1. Check Constraints

A check constraint is the most generic constraint type. It allows you to specify that the value in a certain column must satisfy a Boolean (truth-value) expression. For instance, to require positive product prices, you could use:

CREATE TABLE products (
    product_no integer,
    name text,
    price numeric CHECK (price > 0)
);

```

Doesn't this mean `CHECK` is one kind of constrains, and `UNIQUE` is another kind of constrain?

Is `check` in `check_unique` a verb? Or do I miss anything?